controllers/btcrate-controller.js

```diff
@@ -0,0 +1,18 @@
+const btcRateService = require('../service/btcrate-service');
+
+class BTCRateController {
+
+    async btcRate(req, res, next){
+        try {
+            const userData = await btcRateService.btcRate();
+
+            console.log(userData);
+
+            res.send(userData);
+        } catch (e) {
+            next(e);
+        }
+    }
+}
+
+module.exports = new BTCRateController();
```

Comment:
Here you are exporting an instance of the class - all dependant files are using this single object. Therefore it's either not necessary to create a separate class (you can declare `const BTCRateController = { ... }` and `module.exports = { BTCRateController };`) or export the entire class (`module.exports = { BTCRateController };`

Reply:
May you explain me the problem of using singletone?

Reply:
May you explain me what is the problem of using singletone?